po/zh-TW.po

```diff
@@ -8145,12 +8145,12 @@ msgstr ""
 
 #: src/traits.md:1
 msgid "# Traits"
-msgstr ""
+msgstr "# 特徵"
 
 #: src/traits.md:3
 msgid ""
 "Rust lets you abstract over types with traits. They're similar to interfaces:"
-msgstr ""
+msgstr "Rust 可讓您依據特徵對類型進行抽象化處理，這與介面相似："
```

Comment:
type is already translated as 型別 instead of 類型 https://github.com/google/comprehensive-rust/blob/main/po/zh-TW.po#L14856

We should avoid inconsistencies.